pkg/istiovalues/filps_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package istiovalues
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/istio-ecosystem/sail-operator/pkg/helm"
+)
+
+func TestApplyFipsValues(t *testing.T) {
+	tests := []struct {
+		name         string
+		expectValues helm.Values
+		expectErr    bool
```

Comment:
I'd prefer a `fipsEnabled bool` in the testCase struct that controls the writing of the tmp file instead of just enabling fips after the first test because I think we shouldn't be making assumptions about the order of test executions